manager/readit.py

```diff
@@ -18,71 +18,71 @@ def main(insert, add, tag, delete, clear, update, search, view, version):
     """
     Readit - Command-line bookmark manager tool.
     """
-    d = db.DatabaseConnection()
+    object_database = database.DatabaseConnection()
 
```

Comment:
I think we should call this as `database_object` or `database_connection`

Reply:
@bhavin192 
Yes, database_connection can be better.